Health.py

```diff
@@ -1,39 +1,102 @@
-# For Healing:
-# vitC heals health
-# vitB heal magic
-# supplies repair mechanical
+# Dictionary of food items and their attributes: [weight, health]
+FoodDict = {
+    "Bread": [10, 0],
+    "Apple": [5, 0],
+    "Chicken Head": [20, 20],
+    "Thorny Fruit": [-20, -20],
+}
 
-# food item would be ['foodName', 'food', [vitC_Value, vitB_Value]]
+# Dictionary of repair items and their attributes: [Mechanical]
+RepairItems = {"Wrench": 10, "Toolkit": 30, "Repair Box": 100}
 
-# Enemy has health/magic/mechanical meters, attack & defense profile, Loot
-# If mechanical or health are zero, you cannot attack
-# If magic is zero, you may not use attacks with more than 5 magic damage
-# Defense profile is additive and subtracts from attack
-# Only one attack at a time, alternating turns
+# Dictionary of player's statistics: Health, Magic, Mechanical
+PlayerStats = {"Health": 100, "Magic": 100, "Mechanical": 100}
 
-foodList = [["bread", 10, 0], ["apple", 5, 0], ["chicken head", 20, 20]]
-# potions = [['small red potion', 15], ['medium red potion', 30], ['big red potion', 50]]
 
-meterArray = [["health", 100], ["magic", 100], ["supply", 100]]
+# Function to heal the player
+def Heal(foodDict, meter):
+    """
+    Function to heal the player.
 
+    :param food_dict: Dictionary of food items and their attributes.
+    :param meter: Dictionary of player's statistics.
+    """
+    # Starting values for the health and magic meters
+    MAX_HEALTH_VALUE = 100
+    MAX_MAGIC_VALUE = 100
 
-def heal(
-    foodChoice, meter
-):  # searches food items for needed nutritional value then consumes them for points, changes health meter accordingly
-
+    # Display available healing items
     print("Choose a healing item:")
-    for item in foodList:
-        print(f"{item[0]}: {item[1]} health points")
-        selectedItem = input("Item: ")
+    for item, values in foodDict.items():
+        print(f"{item}: {values[0]} health points and {values[1]} Magic points")
+    for item, values in RepairItems.items():
+        print(f"{item}: {values[0]} Mechanical points")
```

Comment:
`TypeError: 'int' object is not subscriptable`

Reply:
resolved